docs/faq.md

```diff
@@ -5,65 +5,67 @@ description: Get to know more about Next.js with the frequently asked questions.
-  <p>It's up to you. You can use the <a href="https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch">fetch API</a> or <a href="https://swr.vercel.app/">SWR</a> inside your React components for remote data fetching; or use our <a href="/docs/basic-features/data-fetching/overview.md">data fetching methods</a> for initial data population.</p>
+  <summary>Can I use Next.js with my favorite JavaScript library?</summary>
+  <p>Yes! We have hundreds of examples of this in action in our <a href="https://github.com/vercel/next.js/tree/canary/examples">examples directory</a>.</p>
 </details>
 
 <details>
-  <summary>Can I use it with GraphQL?</summary>
-  <p>Yes! Here's an <a href="https://github.com/vercel/next.js/tree/canary/examples/with-apollo">example with Apollo</a>.</p>
+  <summary>Can I use Next.js with GraphQL?</summary>
+  <p>Yes! Here's an <a href="https://github.com/vercel/next.js/tree/canary/examples/with-apollo">example with Apollo</a> and an <a href="https://github.com/vercel/next.js/tree/canary/examples/api-routes-graphql">example API route with GraphQL</a>.</p>
 </details>
 
 <details>
-  <summary>Can I use it with Redux?</summary>
-  <p>Yes! Here's an <a href="https://github.com/vercel/next.js/tree/canary/examples/with-redux">example</a>. And there's another <a href="https://github.com/vercel/next.js/tree/canary/examples/with-redux-thunk">example with thunk</a>.</p>
+  <summary>Can I use Next.js with Redux?</summary>
+  <p>Yes! Here's an <a href="https://github.com/vercel/next.js/tree/canary/examples/with-redux">example with Redux</a> and an <a href="https://github.com/vercel/next.js/tree/canary/examples/with-redux-thunk">example with thunk</a>.</p>
+</details>
+
+<details>
+  <summary>Can I make a Next.js Progressive Web App (PWA)?</summary>
+   <p>Yes! Here's our <a href="https://github.com/vercel/next.js/tree/canary/examples/progressive-web-app">Next.js PWA Example</a>.</p>
 </details>
 
 <details>
   <summary>Can I use a CDN for static assets?</summary>
-  <p>Yes. You can read more about it <a href="/docs/api-reference/next.config.js/cdn-support-with-asset-prefix.md">here</a>.</p>
+  <p>Yes! You can read more about how to do it <a href="/docs/api-reference/next.config.js/cdn-support-with-asset-prefix.md">here</a>.</p>
 </details>
 
 <details>
-  <summary>Can I use Next with my favorite JavaScript library or toolkit?</summary>
-  <p>Since our first release we've had many example contributions. You can check them out in the <a href="https://github.com/vercel/next.js/tree/canary/examples">examples</a> directory.</p>
+  <summary>How can I change the internal webpack configs?</summary>
+  <p>Next.js tries its best to remove the overhead of webpack configurations through supporting the most popular use cases. For advanced cases where more control is needed, refer to the <a href="/docs/api-reference/next.config.js/custom-webpack-config.md">custom webpack config documentation</a>.</p>
```

Comment:
```suggestion
  <p>In most cases, no manual webpack configuration is necessary since Next.js automatically configures webpack. For advanced cases where more control is needed, refer to the <a href="/docs/api-reference/next.config.js/custom-webpack-config.md">custom webpack config documentation</a>.</p>
```